frontend/src/components/table/TableItem.js

```diff
@@ -228,7 +228,7 @@ class TableItem extends PureComponent {
    */
   _editProperty = ({ e, property, focus, item, select, mark }) => {
     if (item ? !item.readonly : true) {
-      if (this.state.edited === property && e) e.persist(); // e.stopPropagation();
+      if (this.state.edited === property && e) e.persist();
```

Comment:
My comment was about `persist`, not `stopPropagation`

Reply:
"If you want to access the event properties in an asynchronous way, you should call event.persist() on the event, which will remove the synthetic event from the pool and allow references to the event to be retained by user code."  - With other words if we do a setState down below using the event and we need data from it we use that
![Screenshot 2020-08-18 at 07 54 24](https://user-images.githubusercontent.com/1708561/90472037-3d344a80-e128-11ea-84c2-c75518c833a1.png)

If you do otherwise you'll get nullified event props.

<img width="835" alt="Screenshot 2020-08-18 at 07 57 27" src="https://user-images.githubusercontent.com/1708561/90472160-8f756b80-e128-11ea-90f8-75b9c6f97195.png">